babel.config.json

```diff
@@ -236,7 +236,7 @@
           "@department-of-veterans-affairs/platform-forms/fields": "./src/platform/forms/fields.js",
           "@department-of-veterans-affairs/platform-forms/helpers": "./src/platform/forms/helpers.js",
           "@department-of-veterans-affairs/platform-forms/sub-task": "./src/platform/forms/sub-task/index.js",
-          "@department-of-veterans-affairs/platform-forms/save-in-progress": ".src/platform/forms/save-in-progress/SaveInProgressIntro",
+          "@department-of-veterans-affairs/platform-forms/save-in-progress": "./src/platform/forms/save-in-progress/SaveInProgressIntro.jsx",
```

Comment:
I had to wade into this file in order to satisfy the linter.  Turns out there was a couple typos here.